DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -141,8 +144,11 @@ const char* const TaggingVariableTokens[] = {
   /* [jetPt]                                    = */ "jetPt",
```

Comment:
The comment is for the TaggingVariableDescription array above. There is no description for , e.g. 
Jet_SoftMu which should be right below z_ratio on L135